docs/src/prototypes/chatMessages/ChatMessageWithPopover/Popover.tsx

```diff
@@ -4,6 +4,9 @@ import cx from 'classnames'
 
 export interface PopoverProps {
   className?: string
+  onForceShowActionMenuChange?: (val: boolean) => void
+  onShowActionMenuChange?: (val: boolean) => void
+  chatMessageRef?: HTMLElement
```

Comment:
may suggest to rename it to `chatMessageElement` - because, if it is a ref as announced by prop, as a client I would expect Ref interface to be applicable: `chatMessageRef.current`. Here, in contrast, this prop provides element directly, not a ref on element